float64 pos_n
float64 pos_e
float64 pos_d
float64 vel_n
float64 vel_e
float64 vel_d
